docs/examples/Track2Track_Fusion_Tutorial.py

```diff
@@ -0,0 +1,876 @@
+        meas_fusion_tracker.detector = DummyDetector(current=detections)
+        meas_fusion_tracker.__iter__()
+        _, tracks = next(meas_fusion_tracker)
+        meas_fusion_tracks.update(tracks)
+
+    # Run the GM-PHD for track fusion. Similar to the measurement fusion, this tracker gets run
+    # twice, once for each set of tracks.
+    for tracks_as_meas in [sensor1_tracks, sensor2_tracks]:
+        dummy_detector = DummyDetector(current=[time, tracks_as_meas])
+        track_fusion_tracker.detector = Tracks2GaussianDetectionFeeder(dummy_detector)
+        track_fusion_tracker.__iter__()
+        _, tracks = next(track_fusion_tracker)
+        track_fusion_tracks.update(tracks)
+
+    # ----------------------------------------------------------------------
+
+    # Add ground truth data to metric managers
+    truths = radar_simulator.groundtruth.current
+    for manager in [sensor1_mm, sensor2_mm, meas_fusion_mm, track_fusion_mm]:
+        manager.add_data(groundtruth_paths=truths[1], overwrite=False)
+
+    # Add measurements to metric managers
+    sensor1_mm.add_data(detections=s1d[1], overwrite=False)
+    sensor2_mm.add_data(detections=s2d[1], overwrite=False)
+    meas_fusion_mm.add_data(detections=s1d[1], overwrite=False)
+    meas_fusion_mm.add_data(detections=s2d[1], overwrite=False)
+    track_fusion_mm.add_data(detections=s1d[1], overwrite=False)
+    track_fusion_mm.add_data(detections=s2d[1], overwrite=False)
+
+
+# Ensure that all tracks have been extracted from the trackers
+jpda_tracks.update(jpda_tracker.tracks)
+gmlcc_tracks.update(gmlcc_tracker.tracks)
+meas_fusion_tracks.update(meas_fusion_tracker.tracks)
+track_fusion_tracks.update(track_fusion_tracker.tracks)
+
+# Remove tracks that have just one state in them as they were probably from clutter
+jpda_tracks = set([track for track in jpda_tracks if len(track) > 1])
+gmlcc_tracks = set([track for track in gmlcc_tracks if len(track) > 1])
+meas_fusion_tracks = set([track for track in meas_fusion_tracks if len(track) > 1])
+track_fusion_tracks = set([track for track in track_fusion_tracks if len(track) > 1])
+
+# Add tracks to metric managers
+sensor1_mm.add_data(tracks=jpda_tracks, overwrite=False)
+sensor2_mm.add_data(tracks=gmlcc_tracks, overwrite=False)
+meas_fusion_mm.add_data(tracks=meas_fusion_tracks, overwrite=False)
+track_fusion_mm.add_data(tracks=track_fusion_tracks, overwrite=False)
+
+# %%
+# 10: Plot the Results
+# --------------------
+# Next, we will plot all of the resulting tracks and measurements. This will be done in two plots.
+# The first plot will show all of the data, and the second plot will show a closer view of one
+# resultant track.
+#
+# These plots are done in 2D to make them more readable. They are still very busy, so you are
+# invited to run it yourself and remove some of the items from the plot. Or, if you are running
+# the notebook locally, add the next line to your code to make it an interactive plot.
+#
+# %matplotlib widget
```

Comment:
Is this needed in the docs?

Reply:
Maybe not? I added it as an excuse for the very messy plot, it is a lot easier to see using the widget mode. But I can remove it if you think it doesn't quite fit!

Reply:
I've changed the wording a little. I think it is still useful to have in an example like this. I'm hoping it will encourage users to explore the plots and results more closely on their own, as that is the best way to fully understand the material.

Reply:
No it's okay, if it's needed for the plotting leave it in - I just wasn't sure if it did anything on the documentation version